tools/packaging/guest-image/build_se_image.sh

```diff
@@ -9,27 +9,18 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly script_name="$(basename "${BASH_SOURCE[0]}")"
-readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-readonly packaging_root_dir="$(cd "${script_dir}/../" && pwd)"
-readonly kata_root_dir="$(cd "${packaging_root_dir}/../../" && pwd)"
+script_name="$(basename "${BASH_SOURCE[0]}")"
```

Comment:
I'm curious why it needed the removal of `readonly`

Reply:
hmm maybe because same variables are set in [tools/packaging/guest-image/lib_se.sh](https://github.com/kata-containers/kata-containers/pull/9951/commits/15b16041850acd7cd744a5a758c5212048820e30#diff-3513f55f212771374656907cdec9a7f2335c41d9903426eb6687dacc307f1510) so it complains they are readonly?

Reply:
Exactly! 😉 They are declared as readonly before sourcing the library. Fortunately, the two files are located in the same directory. We could differentiate those names to prevent this kind of conflict if the library is used in various places, which doesn't seem to be the case here. I need your advise on this. Thanks!